vitest.config.js

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+	test: {
+		environment: 'jsdom',
+		globals: true,
```

Comment:
If updating to use `vitest.setup.ts`
```suggestion
		globals: true,
		setupFiles: ['./vitest.setup.ts'],
```